src/player.h

```diff
@@ -1152,6 +1155,8 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
         int cached_turn;
         tripoint cached_position;
 
+        int fell_asleep_turn;
```

Comment:
And this member is not initialized and it's not stored in saves (the later is probably not so much a problem, as it's only used for "cosmetic" messages, it does not actually affect the game state).

However, reading from an uninitialized variable is technically undefined behavior. Do you want nasal demon? Because that's how you get nasal demons.